dashboard/src/assets/constants/overviewConstants.js

```diff
@@ -1,2 +1,10 @@
 export const START_PAGE_NUMBER = 1;
 export const ROWS_PER_PAGE = 5;
+
+export const USER_FAVORITE = "user.favorite";
+export const DASHBOARD_SEEN = "global.dashboard.seen";
+export const DASHBOARD_SAVED = "global.dashboard.saved";
+export const SERVER_DELETION = "server.deletion";
+export const DATASET_OWNER = "dataset.owner";
+export const DATASET_ACCESS = "dataset.access";
+export const DATASET_CREATED = "dataset.created";
```

Comment:
Your `import` lists are nicely sorted in each case, but the declarations aren't. While this is a minor issue, sorting this might make it just a bit easier to read.

Reply:
Ordered the declarations